<launch>
  <!-- Node launching -->
  <node pkg="mercator_sensor_fusion_ros_pkg" type="ground_truth_publisher" name="ground_truth_publisher" output="screen">
    <!-- Parameters -->
    <param name="robot_names" value="['base_link_38']" />
    <param name="reference_robot_name" value="base_link_0" />
    
    <!-- Topic remappings -->
    <!-- <remap from="/tf" to="/input_tf" />
    <remap from="ground_truth_poses" to="/output_ground_truth_poses" /> -->
  </node>
</launch>
